src/jacobian_maps.jl

```diff
@@ -1,5 +1,15 @@
 # This file is a part of MGVI.jl, licensed under the MIT License (MIT).
 
+function ChainRulesCore.rrule(::typeof(PDMat{T, S}), dim::Int, mat::S, chol::Cholesky{T, S}) where {T<:Real, S<:AbstractMatrix}
```

Comment:
jacobian for mvnormal used to fail otherwise. PDMat constructor required the adjoint.

Reply:
This we can't do, it's type piracy - MGVI doesn't own any of the types in this `rrule`. We'll need to make use a wrapper function or so that we can define the `rrule` on.